lms/templates/api/canvas/oauth2_redirect.html.jinja2

```diff
@@ -0,0 +1,7 @@
+{% extends "lms:templates/base.html.jinja2" %}
```

Comment:
The base template includes a bunch of resources that are not needed by this page. As they are stylesheets rather than scripts that shouldn't affect the load time. On the other hand it does set a title and favicon which might still be useful here.